src/Autonomous.cpp

```diff
@@ -50,7 +51,7 @@ Autonomous::Autonomous(const URCLeg &_target, double controlHz)
 		urc_target(_target),
 		search_target({_target.approx_GPS(0) - PI, _target.approx_GPS(1), -PI / 2}),
 		gate_targets({NAN, NAN, NAN}, {NAN, NAN, NAN}),
-		poseEstimator({1.5, 1.5}, gpsStdDev, Constants::WHEEL_BASE, 1.0 / controlHz),
+		poseEstimator({0.2, 0.2}, gpsStdDev, Constants::WHEEL_BASE, 1.0 / controlHz),
```

Comment:
In the future we might want to have this `{0.2, 0.2}` value as a constant so it is more clear what it's for when reading the code.